frontend/src/component/feature/FeatureStrategy/FeatureStrategyCreate/FeatureStrategyCreate.tsx

```diff
@@ -101,11 +101,24 @@ export const FeatureStrategyCreate = () => {
 
     useEffect(() => {
         if (shouldUseDefaultStrategy) {
-            setStrategy((defaultStrategy as any) || DEFAULT_STRATEGY);
+            const strategyTemplate = defaultStrategy || DEFAULT_STRATEGY;
+            if (
+                strategyTemplate.parameters &&
+                'groupId' in strategyTemplate.parameters &&
+                strategyTemplate.parameters.groupId === '' &&
+                featureId
+            ) {
+                strategyTemplate.parameters.groupId = featureId;
+            }
+            setStrategy(strategyTemplate as any);
```

Comment:
as any can be avoided by changing

```
export interface IFeatureStrategy {
    ...other fields
    title?: string | null;
    disabled?: boolean | null;
}
```
Our FE code didn't match openapi schema so let's adjust them instead of ignorign types

Reply:
I added nullable constraint ont itle and disabled as indicated by the openapi schema

Reply:
Turns out doing the type change opens a huge can of worms. I think more important to fix the functionality. Decided to skip this for now